test/e2e/lib/flows/login-flow.js

```diff
@@ -205,6 +205,7 @@ export default class LoginFlow {
 		}
 
 		if ( this.account.username !== 'e2eflowtestinggutenbergsimpleedge' ) {
+			await StatsPage.overrideABTestInLocalStorage( 'customerHomeAll', 'control' );
```

Comment:
I think the default variants always take precedence in e2e tests. For example, I bumped `customerHomeAll` up to `100` and the tests passed locally for me.

Reply:
Yeah, @racheleldracher confirmed this (p6fDka-HC-p2#comment-1831), so e2e should keep passing until we fully enable Customer Home. Not sure why we had some runs where e2e were using the tested variant (@Automattic/e2e-test-reviewers might help). 

In any case, I don't think it makes sense to assert that we land on Stats or Customer Home after selecting My Sites, so I replaced that assertion with a more generic check that verifies that the My Sites layout is loaded.